setup.py

```diff
@@ -160,6 +160,7 @@
 TESTS_REQUIRE = [
     # test dependencies
     "absl-py",
+    "joblibspark",
```

Comment:
joblib or joblibspark

Reply:
`joblib` is a requirement of `joblibspark`, while both would be used, specifying `joblibspark` should be fine.